OneCommunityGlobal/HighestGoodNetworkApp
# Shihao - hotfix group member add time issue

# Description
<img width="804" alt="Screenshot 2023-10-24 at 15 34 01" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/62367016/1227fddd-aec8-4d11-a38e-535ab66f7196">
<img width="800" alt="Screenshot 2023-10-24 at 15 34 28" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/62367016/8e5a2615-1afc-4d99-b49d-8223e86fae60">

## Related PRS (if any):
This PR fixed the issue of group member added time and optimized the sorting logic in #1342.

## Main changes explained:
- Corrected the issue of group members added time
- Optimized the sorting logic

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin/owner user
4. go to dashboard→ Other Links → Teams 
5. Check if the "addDateTime" is correct. (Hint. You can change your system date and add the member to the team to simulate adding the user at a different time)

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/62367016/336a1c32-6637-4904-9bd6-b4a0794410af
